certbot/docs/using.rst

```diff
@@ -474,29 +474,43 @@ like
 Revoking certificates
 ---------------------
 
-If your account key has been compromised or you otherwise need to revoke a certificate,
-use the ``revoke`` command to do so. Note that the ``revoke`` command takes the certificate path
-(ending in ``cert.pem``), not a certificate name or domain. Example::
+If you need to revoke a certificate, use the ``revoke`` subcommand to do so.
 
-  certbot revoke --cert-path /etc/letsencrypt/live/CERTNAME/cert.pem
+.. note:: Revoking a certificate will have no effect on the rate limit imposed by the Let's Encrypt server.
+
+A certificate may be revoked by providing its name (see ``certbot certificates``) or by providing
+its path directly::
+
+  certbot revoke --cert-name example.com
+
+  certbot revoke --cert-path /etc/letsencrypt/live/example.com/cert.pem
+
+If the certificate being revoked was obtained via the ``--staging``, ``--test-cert`` or a non-default ``--server`` flag,
+that flag must be passed to the ``revoke`` subcommand.
+
+.. note:: By default, Certbot will **delete** the certificate after revoking it. Use ``--no-delete-after-revoke`` to prevent the
+          certificate from being deleted. Note that Certbot will try to renew revoked certificates if they are not deleted.
```

Comment:
To me it seems worthy of documentation because potential (irreversible) pitfalls around deletion and the slightly unintuitive renewal behavior. Just to trigger some thoughts for the reader about what they should do, before they have to do it. 

The latter option of your suggestions good to me, I've tried to do that.